config/exporters/monitoring/grafana/base/kustomization.yaml

```diff
@@ -2,3 +2,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
 - prometheus-exporter-service.yaml
+images:
+- name: exporter
+  newName: quay.io/redhat-appstudio/o11y
+  newTag: latest
```

Comment:
Yesterday they have updated the tag to latest since the next is not used anymore.
[Reference PR](https://github.com/redhat-appstudio/integration-service/commit/ef13f86c6b120157ff6e164c0ed2648f67755e03
)

Reply:
What about possible race conditions? Has this been considered or is it a non-issue?

Reply:
Yes, since this is for local testing I think it's not an issue. We will have different PR in infra-deployments to update the commit-ref which would be done automatically by the release pipeline.